### ISSUE TITLE
When you edit you go back to main page

### ISSUE THREAD

**Issue by Elisagig** [Mergpijp/GlobalOrthodoxy#107]
Dear colleagues,

A couple of my colleagues complained about this on Friday. When they  "edit" and entry, they do not go back to the edit page but end up completely in the beginning. Here is what a colleague sent: 
Habtom "I deleted a double entry on page 3 similar to the prayer book at the beginning of page 1 and the database automatically threw me back to the first page... why is that? I expect to stay at least on page three to go further.....This was repeated when I deleted similar entry on page 7"

**Comment by Mergpijp** [comment]
So this is for uploaded files or for everything? I managed to redirect to referer when deleting files I think that solves one problem.
Do you want to go back to the page your from if you edit a file for example?

**Comment by Mergpijp** [comment]
So this is done deploying latest changes now and merging later for the modal feature.

**Comment by Mergpijp** [comment]
So enlighten me wich deletions need a thrash bin for later recovery? I mean for example: uploadedfiles, authors, translators.

**Comment by Elisagig** [comment]
Yes want to go back to the page we were working on while editing and not completely back to the interface 

Thanks

**Comment by Mergpijp** [comment]
Yes that is done and deployed. But you said: 

Another related question is: How can we retrieve an entry that I wrongly have deleted?

Shall I make uploadedfiles retrievable? I can not think of any other entry worth it.

**Comment by Elisagig** [comment]
Will ask Heleen

**Comment by Mergpijp** [comment]
My suggestion was done in another issue.

**Comment by Elisagig** [comment]
I got that again by colleagues saying: Cannot edit authors while on the create page

**Comment by Elisagig** [comment]
Hi got again messages from 2 colleagues about this issue. They are saying: 
"Save and Add Another, Save and Continue Editing and Save are not working as they should. They throw you to Titles...."
@Woseseltops @Mergpijp

**Comment by Mergpijp** [comment]
So you are suggesting update modals for uploadedfiles?
For me the publication buttons: save and etc are working cant seem to reproduce what you wrote?

**Comment by Elisagig** [comment]
The problem is that once they do save they end up back in titles page and not in the main page where they are working on the content editing

**Comment by Elisagig** [comment]
The other problem is that when you go to "Files" and you do "Save" and then you realize you made an error and want to change again you need to go to "navigate" or "uploaded files" and search for the image name again and then do save

**Comment by Mergpijp** [comment]
The first comment I do not get. For me if i go to city page2 for example and press new publication save will get me back to city page2. The second could be fixed if you make a file and press submit you want to go to the last page of files it now returns to file show.

**Comment by Mergpijp** [comment]
So for your first comment if you press 'save and continue editing' don't you have what you want?

**Comment by Elisagig** [comment]
The issue is happening now with the "Save" button. 
![Screenshot_20210312-173915_Slack](https://user-images.githubusercontent.com/61408472/110970436-0b036800-835a-11eb-9620-e7318202d4ad.jpg)

**Comment by Mergpijp** [comment]
I would like to see the error yes that might help. I dont really get it maybe I need to fill in more?

**Comment by Elisagig** [comment]
Hi @Woseseltops we still have this issue. When we edit and "save" we end up in the main page rather than the page we were working on, so it consumes us more work
